nix/cardano-services/deployments/default.nix

```diff
@@ -211,6 +212,7 @@ in
           blockfrost-worker.enabled = true;
           pg-boss-worker.enabled = true;
           pg-boss-worker.metadata-fetch-mode = "smash";
+          pg-boss-worker.smash-url = "https://preview-smash.world.dev.cardano.org";
```

Comment:
I would say path `/api/v1` is required here as well.

Is there some reason I don't know or is it just a refuse which still needs to be addressed?

Reply:
I only did manual testing of the url provided in slack. Thank you for confirming it needs the /api/v1